geospatial-utils/main.py

```diff
@@ -34,9 +35,15 @@ def main():
 
     if args.command == "convert":
         logger.debug(f"Converting {args.input_url} to {args.output_file}")
+        source_url = args.input_url
         source = fileutils.get(args.input_url, int(args.ttl))
         logger.debug(f"Local input copy: {source.absolute()}")
-        # destination = pathlib.Path(args.output_file)
+
+        data = ed269.load(source)
+        destination = pathlib.Path(args.output_file)
+        destination.write_text(json.dumps(data, indent=2))
```

Comment:
I know this is just one step toward the final result, but it could be very surprising if someone tried to use this tool in its current form and found that it wrote ED-269 data rather than ED-318 data -- perhaps there could be a warning log message indicating that conversation to ED-318 is not yet implemented/was not performed/etc?